src/test/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCallerTest.scala

```diff
@@ -560,4 +574,18 @@ class VanillaUmiConsensusCallerTest extends UnitSpec with OptionValues {
     source.cigar.toString() shouldBe "3M"
     source.sam shouldBe Some(rec)
   }
+
+  it should "not trim reads with an insert size shorter than the read length when there's an insertion in the middle" in {
```

Comment:
this is the new test to confirm the new behavior.  Previously, the insert size would be calculated as 80bp, so the consensus read would have been 80bp.  This happened because the insert size doesn't count inserted bases in the insert.  The correct length of the consensus bases is 100bp.